pkg/cover/store.go

```diff
@@ -264,9 +264,13 @@ func (l *memoryStore) Get(name string) []string {
 
 // Get returns all the registered service information
 func (l *memoryStore) GetAll() map[string][]string {
+	res := make(map[string][]string)
 	l.mu.RLock()
 	defer l.mu.RUnlock()
-	return l.servicesMap
+	for k,v := range l.servicesMap {
+		res[k] = append(make([]string,0,len(v)), v...)
```

Comment:
suggest to just use 
`res[k]=v`

Reply:
`res[k]=v` is not totally safe if the caller modify the slice with index

Reply:
v is not a pointer of the actual value of `l.servicesMap`. FYI: https://github.com/golang/go/wiki/Range#gotchas

Reply:
@CarlJi, v is slice, it is sth like pointer